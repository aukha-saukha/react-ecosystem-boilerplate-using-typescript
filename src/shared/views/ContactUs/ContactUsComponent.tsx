import * as React from 'react';

const ContactUsComponent: React.FC<{}> = (): JSX.Element => (
  <div className="font-size-16" data-test="main">
    Contact us page
  </div>
);

export default ContactUsComponent;
