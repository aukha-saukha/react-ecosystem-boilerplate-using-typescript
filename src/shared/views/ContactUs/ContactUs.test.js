import { shallow } from 'enzyme';
import React from 'react';

import ContactUs from './ContactUsComponent';

describe('"Contact Us" component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContactUs />);

    expect(wrapper).toMatchSnapshot();
  });
});
