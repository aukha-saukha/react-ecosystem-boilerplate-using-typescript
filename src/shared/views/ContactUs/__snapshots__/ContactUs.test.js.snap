// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"Contact Us" component renders correctly 1`] = `
<div
  className="font-size-16"
  data-test="main"
>
  Contact us page
</div>
`;
